tests/science/science.cfg

```diff
@@ -1 +1,21 @@
-[test]
+[science_test_snotel]
+driver = classic
+test_description = snotel point simulations
+global_parameter_file = global_param.classic.5.0_snotel.txt
+expected_retval = 0
+check = not_implemented
+plots = water_year
+observations_path = snotel
+vic4.2.d = vic_4_2
+vic5.0.0 = vic_5_0
```

Comment:
I'd like to rethink this. What do you think about a list of previous simulations that we will compare to. Something like:

```
[science_test_snotel]
...
observations_path = snotel
...
[[compare]]
[[[VIC.4.2.d]]]
version = 4.2
archive = archive/vic_4_2/
[[[VIC.5.0.0.rc1]]]
version = 5.0
archive = archive/vic_5_0_rc1/
```
This will allow us to arbitrarily add in additional compare tests without having to change `run_tests.py`.